recipe/meta.yaml

```diff
@@ -129,7 +143,7 @@ requirements:
     - libxslt {{ libxslt }}        # [win]
     # Needs to be in hosts to allow QtPdf to build but is really a build dependency so no need to add to runtime deps.
     - html5lib 1.1
-    - libtiff {{ libtiff }}
```

Comment:
You could just put this in the local cbc.yaml so it's a little bit more obvious that it needs to be changed once the base cbc.yaml is updated